packages/vue-instantsearch/src/components/Configure.js

```diff
@@ -21,7 +21,7 @@ export default {
   computed: {
     widgetParams() {
       return {
-        searchParameters: this.$attrs,
+        searchParameters: { ...this.$attrs },
```

Comment:
I'm not 100% sure but I think our build system does not allow destructuring for vue. Would this work as an alternative?

```suggestion
        searchParameters: Object.assign({}, this.$attrs),
```

Reply:
Indeed 😪